pyproject.toml

```diff
@@ -14,18 +16,41 @@ click = "^8.1"
 jsonschema = "^3.0.0"
 cffconvert = "^2.0.0"
 
-[tool.poetry.dev-dependencies]
+# Packages for developers
+[tool.poetry.group.dev.dependencies]
 pytest = "^7.1.1"
 pytest-cov = "^3.0.0"
+taskipy = "^1.10.3"
+
+# Packages for developers for creating documentation
+[tool.poetry.group.docs]
+optional = true
+
+[tool.poetry.group.docs.dependencies]
 Sphinx = "^4.5.0"
-myst-parser = "^0.17.2"
+myst-parser = "^0.18.0"
+sphinx-autobuild = "^2021.3.14"
+sphinx-book-theme = "^0.3.3"
+# Sphinx - Additional modules
+sphinxcontrib-apidoc = "^0.3.0"
+sphinxcontrib-contentui = "^0.2.5"
+sphinxcontrib-images = "^0.9.4"
+sphinx-icon = "^0.1.2"
+sphinx-favicon = "^0.2"
+sphinxemoji = "^0.2.0"
+sphinxext-opengraph = "^0.6.3"
 
 [tool.poetry.plugins.console_scripts]
 haggis = "hermes.cli:haggis"
 
 [tool.poetry.plugins."hermes.harvest"]
 cff = "hermes.commands.harvest.cff:harvest_cff"
 
+[tool.taskipy.tasks]
+docs-build = "poetry run sphinx-build -M html docs/source docs/build -W"
+docs-clean = "poetry run sphinx-build -M clean docs/source docs/build"
+docs-live = "poetry run sphinx-autobuild docs/source docs/build"
+
 [build-system]
-requires = ["poetry-core>=1.0.0"]
+requires = ["poetry-core>=1.2.0"]
```

Comment:
**question:** Does `taskipy` require this?

Reply:
Nope, this is necessary to have proper dependency groups support. The docs group for example can now be installed with `poetry install --with docs`. This functionality became stable in v1.2 release of `poetry`. Enabling poetry use with RTD and reusing parties don't trip over this by enforcing the version.